### PR TITLE
MF-385: Rewrite patient banner based on carbon designs

### PR DESCRIPTION
**Pull request by denniskigen**
Ticket: https://issues.openmrs.org/browse/MF-385
Zeplin board: https://zpl.io/awvWeKN

This PR lays the groundwork for adopting the new `patient summary` designs.
 
It: 

- Refactors the patient banner to use the new designs.
- Extracts related address logic into a ContactDetails component.

Relates to: https://github.com/openmrs/openmrs-esm-patient-chart/pull/1

Screenshot 2020-11-27 at 09 50 33

Screenshot 2020-11-27 at 09 50 51

### CODE REVIEW

**Review by FlorianRappl — APPROVED** [review]
LGTM! Thanks :beers:

**Comment by denniskigen** [comment]
@FlorianRappl update: patient banner is now available as an extension.

File: package.json [hunk 1]
```diff
@@ -45,6 +45,8 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.29",
     "@fortawesome/free-solid-svg-icons": "^5.13.1",
     "@fortawesome/react-fontawesome": "^0.1.11",
+    "@openmrs/esm-context": "^3.1.4",
```
Comment (FlorianRappl) [inline package.json]: Context does not exist any more. Also use the 3.1.5 version.

Maybe pull back from master first.

Comment (denniskigen) [reply package.json]: Oops, bad rebase. Fixing shortly.

**Review by FlorianRappl — APPROVED** [review]
This is great. Thanks @denniskigen !

**Comment by denniskigen** [comment]
> This is great. Thanks @denniskigen !

Thanks @FlorianRappl. I imagine we hold off on merging this until after the conference?

**Comment by FlorianRappl** [comment]
> I imagine we hold off on merging this until after the conference?

As you like. If you ask me we could merge it in directly. Then I can use it in my video.

**Comment by denniskigen** [comment]
Sure @FlorianRappl, let's merge this. I've made a small [PR](https://github.com/openmrs/openmrs-esm-patient-chart/pull/191) on the patient chart side fixing the CSS so it looks congruent with the old UI.

**Pull request merged**